src/include/network/routines.rb

```diff
@@ -928,23 +928,29 @@ def GetAllInterfaces
     end
 
     def SetLinkUp(dev_name)
+      log.info("Setting link up for interface #{dev_name}")
       Run("ip link set #{dev_name} up")
     end
 
     def SetLinkDown(dev_name)
+      log.info("Setting link down for interface #{dev_name}")
       Run("ip link set #{dev_name} down")
     end
 
     def SetAllLinksUp
       interfaces = GetAllInterfaces()
-      ret = !interfaces.empty?
 
-      interfaces.each do |ifc|
-        Builtins.y2milestone("Setting link up for interface %1", ifc)
-        ret = SetLinkUp(ifc) && ret
-      end
+      return false if interfaces.empty?
 
-      ret
+      interfaces.all? { |i| SetLinkUp(i) }
+    end
+
+    def SetAllLinksDown
+      interfaces = GetAllInterfaces()
+
+      return false if interfaces.empty?
+
+      interfaces.all? { |i| SetLinkDown(i) }
```

Comment:
Again, the code explicitly tries to return a boolean value which is

1. not documented
2. not used by any caller

I hate this!

In this particular case, I think the return value does not even make sense.